### DISCUSSION TITLE
Appearing the output text two times instead of one

### DISCUSSION THREAD

**Discussion by renovatio118** [ImageMagick/ImageMagick#7100] [Help] [answered]
### ImageMagick version

 7.0.9-21

### Operating system, version and so on

Windows

### Description

I'm trying to add text "9999" at the bottom center of one image and behind the text i need white background. 
After many tests i came up with this command:
`magick convert 3071.jpg -size 300x50 xc:white -gravity center -pointsize 36 -annotate +0+0 "9999" -gravity South -composite "C:/bb/3071_9999.jpg"`

but this adds text "9999" in the center of the image as well. 
What is wrong with my command? I'm really confused.

Can anyone please help?

**Comment by snibgo** [comment, marked as answer]
You create two images. One is from `3071.jpg`, the other is from `xc:`. Then you annotate both images. Then you composite the images together, so you will have two annotations.

Perhaps you intended to annotate only the second image. If so, you can do that with parentheses:
```
magick convert 3071.jpg ( -size 300x50 xc:white -gravity center -pointsize 36 -annotate +0+0 "9999" ) -gravity South -composite "C:/bb/3071_9999.jpg"
```

**Reply by renovatio118** [reply]
OMG, now that i'm reading your answer that is so obvious, yet i couldn't get it before. Thank you so much for your help, i really appreciate it.
